static/elements/chromedash-callout.js

```diff
@@ -0,0 +1,136 @@
+import {LitElement, css, html} from 'lit-element';
+
+class ChromedashCallout extends LitElement {
+  static get properties() {
+    return {
+      signedin: {type: Boolean},
+      targetid: {type: String},
+      // TODO(jrobbins): Support sides other than "south".
+      side: {type: String}, // "north", "south", "east", or "west"
+      hidden: {type: Boolean},
+      top: {type: Number},
+      left: {type: Number},
+      cue: {type: String}, // String to send to the server when dismissed.
+      dismissedcues: {attribute: false},
+    };
+  }
+
+  constructor() {
+    super();
+    this.side = 'south';
+    this.hidden = true;
+    this.top = 0;
+    this.left = 0;
+    this.signedin = false;
+    this.dismissedcues = [];
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+    if (this.dismissedcues.includes(this.cue)) {
+      // Don't show the cue because the user has already dismissed it.
+      return;
+    }
+    let targetEl = this.parentNode.querySelector('#' + this.targetid);
```

Comment:
```
try {
  this.attach(this.parentNode.querySelector('#' + this.targetid));
} catch (error) {
   console.error('Failed to attach target', error);
}
```

1. querySelector can throw an error when this.target is invalid
2. nit: use console.error.

Reply:
Done.  Thanks.